turbonomic/operands/ibm-opencontent-flink-operand-orm.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: devops.turbonomic.io/v1alpha1
+kind: OperatorResourceMapping
+metadata:
+  name: flink-jobmanager-orm
+  namespace: $namespace
```

Comment:
Have we standardized the namespace placeholder?

Reply:
Unofficially, everyone so far has it as `$namespace`. But let's clarify this now to avoid confusion, and to make the script easier.

Some suggestions would be:
- `{{ placeholder_ns }}`
- `__placeholder_ns__`
- `<placeholder_ns>`

I prefer the first one `{{ }}` because it's used in other areas with templating, such as go templates.